backend/app/crud/prompt_version.py

```
@@ -0,0 +1,263 @@
+    stmt = (
+        select(PromptVersion)
+        .where(PromptVersion.prompt_id == prompt_id, PromptVersion.is_deleted == False)
+        .order_by(PromptVersion.version.desc())
+        .offset(skip)
+        .limit(limit)
+    )
+
+    return session.exec(stmt).all()
+
+
+def get_prompt_versions_count(session: Session, prompt_id: int, project_id: int) -> int:
```

Ruff (0.12.2)

Unused function argument: `project_id`

(ARG001)